apps/console/src/features/oidc-scopes/pages/oidc-scopes-edit.tsx

```diff
@@ -442,7 +452,7 @@ const OIDCScopesEditPage: FunctionComponent<RouteComponentProps<OIDCScopesEditPa
                                     <Form
                                         id={ FORM_ID }
                                         uncontrolledForm={ false }
-                                        onSubmit={ (values): void => {
+                                        onSubmit={ (values: any): void => {
```

Comment:
Shall we replace typedef `any` here? If it can be an object, we can go with `Record<string, valueType | string`

Reply:
The Form component[1] used here is generic so it expects `onSubmit` function params to be of `any` type, doesn't it?  

[1] https://github.com/wso2/identity-apps/blob/master/modules/form/src/components/form.tsx#L179

Reply:
Fixed with 3f70271